src/applications/vaos/utils/constants.js

```diff
@@ -304,7 +304,7 @@ export const DISTANCES = [
 
 export const REASON_MAX_CHARS = {
   request: 100,
-  direct: 150,
+  direct: 100,
```

Comment:
Did you verify that this fixes the issue? This property is only supposed to be used for the message that goes into the booking notes for direct scheduled appointments. For requests, the `request` property is the limit that our code should be using.

Reply:
I was unable to reproduce the issue in the staging environment. I have traced this message this message in Sentry and the var_resources code base. I'm checking with the backend team about the message length. There appears to be no distinction in message length based on appointment type it the code.

Reply:
I don't believe we're using the 150 `direct` limit at all right now, since direct scheduling is off.

I'm not able to reproduce this issue either, though I can force a similar error from vets-api by putting only spaces in the text box. We should fix that, but it doesn't solve this particular error message. So I wonder if there's some kind of vets-api issue here.

Reply:
That's interesting. I would guess that some extra characters are being added when sending the request. Maybe with the translation of newline/return? Just thinking out loud. I think the backend needs to change regardless since there is an 100 character limit for both direct and request appointment messages.